code/modules/mob/living/carbon/human/human.dm

```diff
@@ -80,7 +80,25 @@
 		var/datum/attribute/atr = attributes[atrname]
 		dat += "[atr.name] [get_attribute_text_level(atr.get_level())]: [round(atr.level)]/[round(atr.level_limit)] + [round(atr.level_buff)]"
 
-	var/datum/browser/popup = new(viewer, "skills", "<div align='center'>Attributes</div>", 300, 300)
+	dat += ""
+	for(var/atrname in attributes)
+		var/datum/attribute/atr = attributes[atrname]
+		var/bonusname = null
+		var/bonus_amount = 0 //We want to add 100 to health/sanity
+		switch(atrname)
+			if("Fortitude")
+				bonusname = "Max Health"
+				bonus_amount = 100
+			if("Prudence")
+				bonusname = "Max Sanity"
+				bonus_amount = 100
+			if("Temperance")
+				bonusname = "Work Speed"
+			if("Justice")
+				bonusname = "Movement Speed"
+		dat += "[bonusname] : [round(atr.get_level()) + bonus_amount] + [round(atr.level_bonus)]"
```

Comment:
This feels like a very hacky way to do it...

Reply:
It IS, but it is fine for now. Someday down the line I might separate work speed and movement speed into work speed/work chance and movement speed/force but this works for the current system.